### PR TITLE
Make small comment adjustment in e2e test

### PR DESCRIPTION
**Pull request by noahtallen**
This PR will test that the new docker image trigger (#81239) works correctly on trunk with e2e-test-only changes.

### CODE REVIEW

File: packages/calypso-e2e/src/lib/blocks/block-flows/slideshow.ts [hunk 1]
```diff
@@ -21,7 +21,6 @@ export class SlideshowBlockFlow implements BlockFlow {
 
 	/**
 	 * Constructs an instance of this block flow with data to be used when configuring and validating the block.
-	 *
```
Comment (noahtallen) [inline packages/calypso-e2e/src/lib/blocks/block-flows/slideshow.ts]: the linter didn't like these extra lines for some reason

Comment (dpasque) [reply packages/calypso-e2e/src/lib/blocks/block-flows/slideshow.ts]: Oh weird, really? My local linter always auto-adds them. 😆 I'm kind of surprised actually!

**Comment by noahtallen** [comment]
Updated this to only modify a file that we know got stuck previously (from this PR: #81151)

File: packages/calypso-e2e/src/lib/blocks/block-flows/slideshow.ts (hunk 1 above)
Comment (noahtallen) [reply packages/calypso-e2e/src/lib/blocks/block-flows/slideshow.ts]: 🤔 that's confusing! I ended up reverting this change to use a file that we know got stuck, so I guess that's a problem for another time 😅

**Pull request merged**